src/samples.jl

```diff
@@ -65,6 +65,15 @@ Returns `a.encoded == b.encoded && a.info == b.info && a.data == b.data`.
 """
 Base.:(==)(a::Samples, b::Samples) = a.encoded == b.encoded && a.info == b.info && a.data == b.data
 
+"""
+    isequal(a::Samples, b::Samples)
+
+Returns `isequal(a.encoded, b.encoded) && isequal(a.info, b.info) && isequal(a.data, b.data)`.
```

Comment:
...i kinda hate this docstring BUT it's how the above docstring was described as well. plz feel free to edit.

Reply:
Could add a bit more in case it helps:
```suggestion
Checks if each field of `a` and `b` are `isequal` to each other; specifically, this function returns `isequal(a.encoded, b.encoded) && isequal(a.info, b.info) && isequal(a.data, b.data)`.
```